src/ensembl/src/content/app/custom-download/containers/content/attributes-accordion/AttributesAccordion.tsx

```diff
@@ -122,7 +122,7 @@ const AttributesAccordion = (props: Props) => {
         </span>
         <span className={styles.resetIcon} onClick={onReset}>
           <ImageButton
-            buttonStatus={Status.ACTIVE}
+            buttonStatus={Status.UNSELECTED}
```

Comment:
Here and below — perhaps default is better?

Maybe the default status should also be styled in ImageButton as having a blue fill then? So that we don't have to write "UNSELECTED", which implies that something can be selected, when the button can exist only in a single state?